### PR TITLE
Odim-5768: Add Implementation for License Service

### PR DESCRIPTION
**Pull request by hemkapoor**
- new service directory with basic RPCs implements for LicenseService
- Add deployment-related changes
- Add UT
- Implement /redfish/v1/LicenseService - GET. Include routes and RPCs in api service
- Implement retrieval logic to store BMC license information in in-memory DB
- Implementation of /redfish/v1/LicenseService/Licenses - GET
- Implement /redfish/v1/LicenseService/Licenses/{LicenseId} - GET
- Implement individual license retrival

### CODE REVIEW

File: .github/workflows/build_unittest.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ jobs:
           go install github.com/golang/protobuf/protoc-gen-go@v1.3.2
           sudo cp $GOPATH/bin/protoc-gen-go /usr/bin
           sudo cp $GOPATH/bin/protoc-gen-go-grpc /usr/bin
-          protos=("account" "aggregator" "auth" "chassis" "events" "fabrics" "managers" "role" "session" "systems" "task" "telemetry" "update" "compositionservice")
+          protos=("account" "aggregator" "auth" "chassis" "events" "fabrics" "managers" "role" "session" "systems" "task" "telemetry" "update" "compositionservice" "license")
           for str in ${protos[@]}; do
```
Comment (jeevan-kamkar) [inline .github/workflows/build_unittest.yml:81]: IMHO it should be `licenses` instead of `license` @amar-shalgar  @shivacharanms  please suggest

**Review by jeevan-kamkar — COMMENTED** [review]
IMHO it should be `licenses` instead of `license` similar to systems , fabrics  @amar-shalgar @shivacharanms please suggest

File: svc-license/lcommon/common.go [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+	"github.com/ODIM-Project/ODIM/lib-utilities/config"
+	"github.com/ODIM-Project/ODIM/lib-utilities/errors"
+	"github.com/ODIM-Project/ODIM/lib-utilities/response"
+	"github.com/ODIM-Project/ODIM/svc-license/model"
+	log "github.com/sirupsen/logrus"
+)
+
+//GetAllKeysFromTable fetches all keys in a given table
+func GetAllKeysFromTable(table string, dbtype common.DbType) ([]string, error) {
+	conn, err := common.GetDBConnection(dbtype)
```
Comment (AngithCJ) [inline svc-license/lcommon/common.go]: We are planning to remove GetDBConnection() from lib-utilities. I think we can use GetDBConnection function in lib-persistence-manager directly. @shivacharanms @amar-shalgar please suggest.

File: lib-dmtf/model/license.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
+//(C) Copyright [2020] Hewlett Packard Enterprise Development LP
```
Comment (chauberahul1993) [inline lib-dmtf/model/license.go]: Update License Year  2022 all newly added file

**Review by AngithCJ — APPROVED** [review]

**Review by chauberahul1993 — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Review by lpooja — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**